README.md

```diff
@@ -72,19 +72,19 @@ vim.api.nvim_set_keymap(
 
 | Insert / Normal  | Action                                                |
 |------------------|-------------------------------------------------------|
-| `<C-f>/f`        | Toggle between file and folder browser                |
-| `<C-y>/y`        | Copy (multi-selected) files or folders to cwd         |
-| `<C-d>/dd`       | Delete (multi-selected) files or folders              |
-| `<C-r>/r`        | Rename (multi-selected) files                         |
-| `<C-e>/e`        | Add File/Folder at cwd; trailing `/` creates folder   |
-| `--/m`           | Move multi-selected files to cwd                      |
-| `<C-h>/h`        | Toggle hidden files                                   |
+| `<A-c>/c`        | Add File/Folder at cwd; trailing `/` creates folder   |
+| `<A-r>/r`        | Rename (multi-selected) files                         |
+| `<A-m>/m`        | Move multi-selected files to cwd                      |
+| `<A-y>/y`        | Copy (multi-selected) files or folders to cwd         |
```

Comment:
Shouldn't `cwd` be `path` (or maybe `current path`) for copy, move and create?

Reply:
True, thanks!